docs/examples/CreatableSingle.tsx

```diff
@@ -1,33 +1,6 @@
-import React, { Component } from 'react';
+import React from 'react';
 
 import CreatableSelect from 'react-select/creatable';
-import { ColourOption, colourOptions } from '../data';
-import { ActionMeta, OnChangeValue } from 'react-select';
+import { colourOptions } from '../data';
 
-export default class CreatableSingle extends Component {
-  handleChange = (
-    newValue: OnChangeValue<ColourOption, false>,
-    actionMeta: ActionMeta<ColourOption>
-  ) => {
-    console.group('Value Changed');
-    console.log(newValue);
-    console.log(`action: ${actionMeta.action}`);
-    console.groupEnd();
```

Comment:
I've removed the console logs to simplify the example. I don't think they were necessary for the purpose of this example.